lint/verilator_waiver.vlt

```diff
@@ -18,6 +18,7 @@ lint_off -rule WIDTH -file "*/rtl/ibex_core_tracing.sv" -match "*'RV32M'*"
 lint_off -rule WIDTH -file "*/rtl/ibex_core_tracing.sv" -match "*'RV32E'*"
 lint_off -rule WIDTH -file "*/rtl/ibex_core_tracing.sv" -match "*'BranchTargetALU'*"
 lint_off -rule WIDTH -file "*/rtl/ibex_core_tracing.sv" -match "*'WritebackStage'*"
+lint_off -rule WIDTH -file "*/rtl/ibex_core_tracing.sv" -match "*'SecureIbex'*"
```

Comment:
I wonder if we can actually regex another part of the error message to cover all `bit` parameters?

Reply:
Yes, the regex matches the warning/error message string.

Reply:
Ah yes - which is what you wrote in the original comment!
It doesn't mention anything about parameters in the message:
``Operator VAR 'RV32M' expects 1 bits on the Initial value, but Initial value's CONST '32'h1' generates 32 bits``
We could possibly end up suppressing width mismatches if we do anything to wide. Maybe matching on "VAR" and "Initial value"?

Reply:
Maybe something like that is sufficiently strict but still covers all "assign 32'h0 or 32'h1 to a 1-bit constant":

`-match "Operator VAR '*' expects 1 bits on the Initial value, but Initial value's CONST '32'h?' generates 32 bits"`

Note all of that is strictly optional, feel free to go as far as you'd like to push this topic :smile: